app/models/user.rb

```diff
@@ -1,4 +1,8 @@
 class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
```

Comment:
- [ ] Great work for adding the confirmable but I think the functionality is not working since after sign up and creating a new user I couldn't receive an email to confirm my account so kindly add the functionality while sign up a new user 🔥🚀🙏🏻🙏🏻 and even if it is a new email it shows me this error 🚀🔥✅✅

![image](https://user-images.githubusercontent.com/58553711/217976410-dfcc8ebc-2dfd-4575-b634-85b4bd933344.png)